test/conf/master_store.conf

```diff
@@ -1,10 +1,17 @@
 store {
+    # ip
+    ip = "127.0.0.1"
     # Here is the server port that Dst-Server listens on.
-    listeningPort = 18082
-    # Whether this is the master
-    isMaster = true
+    listeningPort = 8091
```

Comment:
the only difference is the port between master , slave1, slave2 configure. can we simplify the configuration.